cmd/register/register_util.go

```diff
@@ -45,6 +45,7 @@ const registrationVersionPattern = "{{ registration.version }}"
 
 // Additional variable define in fast serialized proto that needs to be replace in registration time
 const registrationRemotePackagePattern = "{{ .remote_package_path }}"
+const registrationDestDirPattern = "{{ .dest_dir }}"
```

Comment:
Can we check if we need a separate flag for this and how the user is going to override it.
Also add examples in the doc in what cases do they use this new flag in combination with remote_package_path

Reply:
Added flags for destinationDir